app/models/manageiq/providers/redhat/infra_manager/provision_workflow.rb

```diff
@@ -54,13 +54,33 @@ def update_field_visibility_linked_clone(_options = {}, f)
   end
 
   def allowed_customization_templates(options = {})
-    if supports_native_clone?
+    source_vm = get_source_vm
+    source = load_ar_obj(source_vm)
+
+    return [] if source.blank?
+    if source_vm.platform.downcase.include?("windows")
+      return allowed_sysprep_customization_templates(source)
```

Comment:
The `return` is unnecessary since there is nothing after the conditional.

Reply:
Done